package.json

```diff
@@ -16,16 +16,17 @@
     "url": "https://github.com/hapi-server/server-nodejs.git"
   },
   "dependencies": {
+    "asyncawait": "^3.0.0",
```

Comment:
Since async/await is not supported, an external package was added to support that functionality